python/ray/dashboard/client/src/api.ts

```diff
@@ -143,15 +143,21 @@ export interface ErrorsResponse {
   }>;
 }
 
-export const getErrors = (hostname: string, pid: string | undefined) =>
-  get<ErrorsResponse>("/api/errors", { hostname, pid: pid || "" });
+export const getErrors = (hostname: string, pid: number | null) =>
```

Comment:
why did we change from undefined to null here? typing issue?

Reply:
Previously the `hostname` and `pid` were being extracted from the URL. Now that we're managing the state within the component, we can use semantically more meaningful types (`number` instead of `string` and `null` instead of `undefined` for the different `pid` options).